src/services/evaluation-result-detail-service.ts

```diff
@@ -75,10 +77,10 @@ export const calculateScore = async (evaluation_result_id: number) => {
     )
 
     const calculated_score =
-      Math.round(Number(evaluationsSum._sum.weighted_score) * 10000) /
-      Math.round(Number(evaluationsSum._sum.weight) * 10000)
+      Math.round(Number(evaluationsSum._sum.weighted_score) * 100) /
```

Comment:
@emmaacha i cant recall we have changes here, can pls correct me cat thanks

Reply:
Ohhh nice catch! 🙌

Reply:
Some commits from the previous release were included, updated it now, thank you!